src/error/metadata/suggestion.rs

```diff
@@ -127,8 +128,7 @@ impl Display for Suggestion {
             Suggestion::Adhoc(msg) => msg.to_string(),
             Suggestion::CheckServerConnection => "Make sure the endpoint is accepting connections and is spelled correctly".to_string(),
             Suggestion::ConvertGraphToSubgraph => "If you are sure you want to convert a non-federated graph to a subgraph, you can re-run the same command with a `--convert` flag.".to_string(),
-
-
+            Suggestion::CheckGnuVersion => "If the command `ldd --version` prints a `glibc` version >= 2.18, you can try installing the Rover binary built for `x86_64-unknown-linux-gnu`.".to_string(),
```

Comment:
How about something like this?

```suggestion
            Suggestion::CheckGnuVersion => "This is likely an issue with your current version of `glibc`. Try running `ldd --version`, and if the version >= 2.18, we suggest installing the Rover binary built for `x86_64-uknown-linux-gnu`".".to_string(),
```